spec/server/update_model_spec.rb

```diff
@@ -43,4 +46,42 @@ def app
     expect(attribute_json["desc"]).to eq("The monster's name")
     expect(attribute_json["display_name"]).to eq("NAME")
   end
+
+  it "does not update attribute options with invalid attribute name" do
+    auth_header(:superuser)
+    json_post(:update_model, {
+      project_name: "labors",
+      actions: [{
+        action_name: "update_attribute",
+        model_name: "monster",
+        attribute_name: improper_name,
+        description: "The monster's name",
+        display_name: "NAME"
+      }]
+    })
+
+    response_json = JSON.parse(last_response.body)
+    expect(last_response.status).to eq(422)
+    expect(response_json['errors'][0]['message']).to eq('Attribute does not exist')
+  end
+
+
+  it "does not update attribute options with the incorrect data type" do
+    auth_header(:superuser)
+    json_post(:update_model, {
+      project_name: "labors",
+      actions: [{
+        action_name: "update_attribute",
+        model_name: "monster",
+        attribute_name: proper_name,
+        description: 123,
+        display_name: "NAME",
+        hidden: 'something'
+      }]
+    })
+
+    response_json = JSON.parse(last_response.body)
+    expect(last_response.status).to eq(422)
+    expect(response_json['errors'][0]['message']).to eq("Update attribute failed")
+  end
 end
```

Comment:
Would it be worth it to add specs that the other `EDITABLE_OPTIONS` are actually editable, or is that too much test fatigue? Or at least a spec around the validation piece that iterates through those?

Reply:
I agree about specing the validation piece, that is trouble.